skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -874,5 +874,45 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestIterPositions(unittest.TestCase):
+    def setUp(self):
+        self.msa_empty = TabularMSA(())
+        self.msa = TabularMSA((DNA('ACGT'), DNA('TGCA')))
+        self.expected = [[Sequence('A'), Sequence('T')],
```

Comment:
I think the implementation I have here is pretty straight-forward. It might be worth looking at if you haven't implemented this already @ebolyen. Took me a little while to figure out how to do it. [Here it is](https://github.com/biocore/scikit-bio/pull/1128/files#diff-53c20fdb722d183bf6a87d56f5925411R940)